accounts/auth.go

```diff
@@ -0,0 +1,63 @@
+package accounts
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/user"
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+type Token struct {
+	token []byte
+}
+
+func Hostname() string {
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Finding Hostname", err)
+	}
+	return hostname
+}
+
+func Username() string {
+	entry, err := user.Current()
+	if err != nil {
+		log.Fatal("Finding Username", err)
+	}
+	return entry.Username
+}
+
+// # of Seconds since ...
+func ExpiryDate() int64 {
+	return time.Now().Add(2 * time.Hour).Unix()
+}
+
+func SecretKey() []byte {
+	return []byte("RandomizeThisPlease")
+}
+
+func NewToken() *Token {
+	token := jwt.New(jwt.SigningMethodHS256)
+
+	claims := token.Claims.(jwt.MapClaims)
+
+	claims["admin"] = false      // TODO: Check this?
+	claims["name"] = Username()  // TODO: Check this?
+	claims["exp"] = ExpiryDate() // TODO: Check this?
+
+	tokenString, err := token.SignedString(SecretKey())
+	if err != nil {
+		fmt.Println("Failed to Sign Token", err, token)
+		panic("Goodbye")
```

Comment:
No `log.Fatal` here?

Reply:
Should be, I had originally started typing in panics, then when I got tired of that I added the Fatal call. In normal circumstances, I would have set up the logging, error handler, and most other common utils before starting to code. I'm a big fan of reuse and consistency. This was just quick and dirty coding ...